src/components/DiffView/index.spec.tsx

```diff
@@ -719,21 +716,25 @@ describe(__filename, () => {
       expect(widget.find(LinterMessage)).toHaveLength(1);
     });
 
-    it('does not render inline comments for deleted lines', () => {
-      const line = nextUniqueId();
+    it('does not render inline comments for uncommentable lines', () => {
+      const _changeCanBeCommentedUpon = jest.fn().mockReturnValue(false);
+      const lineNumber = nextUniqueId();
       const version = createInternalVersion({
         ...fakeVersion,
         id: nextUniqueId(),
       });
       const diff = createDiffWithHunks([
-        createHunkWithChanges([{ type: 'delete', old_line_number: line }]),
+        createHunkWithChanges([
+          { new_line_number: lineNumber, old_line_number: lineNumber },
```

Comment:
I see `type` removed from a lot of these. Any reason? I think it's nice to see the `type` explicitly set because it makes the test easier to read.

Reply:
I did this because whether or not comments are rendered in any of these tests depends on what's returned by `_changeCanBeCommentedUpon`, and not on the `type` at all, so the `type` is irrelevant. I didn't want to pass an explicit `type` because I was worried that someone reading the test would interpret that to mean that the `type` *was* affecting the test outcome.

Does that make sense?

Reply:
ok, that makes sense